src/openklant/components/klantinteracties/api/serializers/internetaken.py

```diff
@@ -70,7 +85,52 @@ class Meta:
             },
         }
 
+    def to_representation(self, instance):
+        response = super().to_representation(instance)
+        response["toegewezen_aan_actor"] = ActorForeignKeySerializer(
+            instance.actoren.order_by("interneactorenthoughmodel__order").first(),
+            context={**self.context},
+        ).data
+
+        response["toegewezen_aan_actoren"] = ActorForeignKeySerializer(
+            instance.actoren.all().order_by("interneactorenthoughmodel__order"),
+            context={**self.context},
+            many=True,
+        ).data
+        return response
+
+    # TODO: remove when depricated actoren field gets removed
+    def _validate_actoren(self):
+        toegewezen_aan_actor = "toegewezen_aan_actor" in self.initial_data
+        toegewezen_aan_actoren = "toegewezen_aan_actoren" in self.initial_data
+
+        if toegewezen_aan_actor == toegewezen_aan_actoren:
```

Comment:
Is it possible? They should have different types since one is iterable and another one is not
What happens if they are not equal? Could you please add a test for it

Reply:
I am basically checking here if the fields got send by the user.
If `toegewezen_aan_actor` is True then the user send it, if not then it is False and the same goes for `toegewezen_aan_actoren`.

This way I can validate if only one of the two fields got used. If both are False then neither were send, and if both are True then both got send.